docs/user-guide/experimental-api.md

```diff
@@ -0,0 +1,30 @@
+# Experimental API
+
+## What is the experimental API
+
+The exerimental API is an API group indended for experimenting with and testing
+APIs. To make this possible, unlike other API groups, there is no guarantee of
+future support, compatibility, or even existence. We do this to lower the
+development costs around making new APIs, to let experiments proceed without
+ongoing maintenance costs.
+
+## Where is the experimental API group?
+
+It is hosted by kube-apiserver, at /apis/experimental/v1alpha1. `kubectl`
+understands experimental objects, so they will work just like other objects in
+the Kubernetes system. You'll have to explicitly say "experimental/" in front of
+resource names, e.g. "experimental/daemonset". Because of the somewhat ephemeral
```

Comment:
I may be behind on this, but how is version negotiation done if not specified in experimental group?